packages/css/src/components/unordered-list/unordered-list.scss

```diff
@@ -36,6 +36,12 @@
   }
 }
 
+.amsterdam-unordered-list--inverse-color,
+.amsterdam-unordered-list--inverse-color:not(.amsterdam-unordered-list--no-markers),
+.amsterdam-unordered-list--inverse-color .amsterdam-unordered-list {
```

Comment:
```suggestion
.amsterdam-unordered-list--inverse-color:not(.amsterdam-unordered-list--no-markers) {
```

Reply:
This will break the stories with more levels. I did not find a solution to pass `args` to `children` and it would seem like something that will go wrong during normal usage.

How likely will it be that there is a nested list of an `inverseColor` list that does not appear in white?